heufybot/modules/commands/pronouns.py

```diff
@@ -0,0 +1,82 @@
+from twisted.plugin import IPlugin
+from heufybot.moduleinterface import IBotModule
+from heufybot.modules.commandinterface import BotCommand
+from heufybot.utils import networkName
+from zope.interface import implements
+import json, os.path
+
+
+class Pronouns(BotCommand):
```

Comment:
I typically like to end my command class names with `Command`. User location is a bit of a bad example since it wasn't originally a command and I never refactored the class name. Sorry for that.